drivers/clock_control/Kconfig.nrf

```diff
@@ -47,14 +47,51 @@ config CLOCK_CONTROL_NRF_K32SRC_EXT_FULL_SWING
 
 endchoice
 
-config CLOCK_CONTROL_NRF_K32SRC_BLOCKING
-	bool "Blocking 32KHz crystal oscillator startup"
-	depends on CLOCK_CONTROL_NRF_K32SRC_XTAL
+config CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
+	bool
+	depends on !SOC_SERIES_NRF91X
+	default y if CLOCK_CONTROL_NRF_K32SRC_RC
+
+if CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
+
+config CLOCK_CONTROL_NRF_CALIBRATION_PERIOD
+	int "Calibration opportunity period (in 250ms units)"
+	default 1
+	range 1 127
+	help
+	  Periodically, calibration action is performed. Action includes
+	  temperature measurement followed by clock calibration. Calibration may
+	  be skipped if temperature change (compared to measurement of previous
+	  calibration) did not exceeded CLOCK_CONTROL_NRF_CALIBRATION_TEMP_DIFF
+	  and number of consecutive skips did not exceeded
+	  CLOCK_CONTROL_NRF_CALIBRATION_MAX_SKIP.
+
+config CLOCK_CONTROL_NRF_CALIBRATION_MAX_SKIP
+	int "Maximum number of calibration skips"
+	default 4
+	range 0 255
 	help
-	  Clock control driver will spin wait in CPU sleep until 32KHz
-	  crystal oscillator starts up. If not enabled, RC oscillator will
-	  initially start running and automatically switch to crystal when
-	  ready.
+	  Calibration is skipped when temperature change since last calibration
+	  did not exceeded configured threshold. If number of consecutive skips
```

Comment:
"~did not exceeded~was less than"

Typo and edge case error (it isn't skipped when the change equaled the threshold).

Reply:
fixed.